src/main/scala/ironoxide/v1/user/Jwt.scala

```diff
@@ -0,0 +1,34 @@
+package ironoxide.v1.user
+
+import com.ironcorelabs.{sdk => jsdk}
+import cats.effect.Sync
+import cats.implicits._
+
+/**
+ * IronCore JWT
+ *
+ * Must be either ES256 or RS256 and have a payload similar to `JwtClaims`, but could be
+ * generated from an external source.
+ *
+ * @param jwt Raw JWT string
+ * @param claims Payload of the JWT
+ * @param algorithm Algorithm used by the JWT
+ */
+case class Jwt private (
+  jwt: String,
+  claims: JwtClaims,
+  algorithm: String
+) {
+  private[ironoxide] def toJava[F[_]](implicit syncF: Sync[F]): F[jsdk.Jwt] =
+    syncF.delay(jsdk.Jwt.validate(jwt))
+}
+
+object Jwt {
+  def apply(j: jsdk.Jwt): Jwt =
+    Jwt(j.getJwt, JwtClaims(j.getClaims), j.getAlgorithm)
+
+  def validate[F[_]](jwt: String)(implicit syncF: Sync[F]): F[Jwt] = {
+    val javaJwt = syncF.delay(jsdk.Jwt.validate(jwt))
```

Comment:
if it were me I wouldn't name the result of the syncF.delay and instead use that nice variable name in the map.

`syncF.delay(jsdk.Jwt.validate(jwt)).map(javaJwt => ...)`

Reply:
Pretty. Done